utils/utils.go

```diff
@@ -49,29 +49,29 @@ func CreateFolder(ctx context.Context, path string, force bool) error {
 
 // CreateFile creates pastel.conf file
 // Print success info log on successfully ran command, return error if fail
-func CreateFile(ctx context.Context, fileName string, force bool) (string, error) {
-	create := func(filename string) error {
-		file, err := os.Create(fileName)
+func CreateFile(ctx context.Context, filePath string, force bool) error {
+	create := func(filePath string) error {
+		file, err := os.Create(filePath)
 		if err != nil {
 			log.WithContext(ctx).WithError(err).Error("Error creating file")
-			return errors.Errorf("failed to create file: %v - err: %v", fileName, err)
+			return errors.Errorf("failed to create file: %v - err: %v", filePath, err)
 		}
 		defer file.Close()
 
-		log.WithContext(ctx).Infof("File created: %s \n", fileName)
+		log.WithContext(ctx).Infof("File created: %s \n", filePath)
 		return nil
 	}
 
 	if force {
-		return fileName, create(fileName)
+		return create(filePath)
 	}
 
 	// create if not already exists
-	if _, err := os.Stat(fileName); os.IsNotExist(err) {
-		return fileName, create(fileName)
+	if _, err := os.Stat(filePath); os.IsNotExist(err) {
```

Comment:
this if can be combined with `if force`

Reply:
Thanks, will improve later in another PR. I also saw some not clean-logic in this function.